shell/common/node_bindings.cc

```diff
@@ -439,14 +433,18 @@ node::Environment* NodeBindings::CreateEnvironment(
     case BrowserEnvironment::kWorker:
       process_type = "worker";
       break;
+    case BrowserEnvironment::kUtility:
+      process_type = "utility_process";
+      break;
   }
 
   v8::Isolate* isolate = context->GetIsolate();
   gin_helper::Dictionary global(isolate, context->Global());
   // Do not set DOM globals for renderer process.
   // We must set this before the node bootstrapper which is run inside
   // CreateEnvironment
```

Comment:
as above, this comment is a bit wrong now (should be "renderer or utility processes") but maybe better to reword it to describe the why and not the how.